scripts/pylint_extensions.py

```diff
@@ -645,19 +624,6 @@ def check_single_constructor_params(self, class_doc, init_doc, class_node):
                 args=(class_node.name,),
                 node=class_node)
 
-    def _handle_no_raise_doc(self, excs, node):
```

Comment:
Removed this function according to this comment: https://github.com/oppia/oppia/pull/7153/files#r304153857